bundles/binding/org.openhab.binding.bticino/src/main/java/be/devlaminck/openwebnet/MonitorSessionThread.java

```diff
@@ -65,7 +65,8 @@ public MonitorSessionThread(OpenWebNet pluginReference, String ipAddress,
 	}
 
 	public void buildEventFromFrame(String frame) {
-		logger.info("Received OpenWebNet frame '" + frame
+		if (logger.isInfoEnabled())
+			logger.info("Received OpenWebNet frame '" + frame

```

Comment:
please add curly braces and change into parameterised statement (see http://www.slf4j.org/faq.html#logging_performance)